CHANGELOG.md

```diff
@@ -102,6 +102,7 @@ Unreleased changes template.
 * (packaging) An empty `requires_file` is treated as if it were omitted, resulting in a valid `METADATA` file.
 * (rules) py_wheel and sphinxdocs rules now propagate `target_compatible_with` to all targets they create.
   [PR #2788](https://github.com/bazel-contrib/rules_python/pull/2788).
+* (pypi) Dependency resolution failures no longer show the full exception traceback in the output
```

Comment:
Please omit this from the changelog. It's nice to have nicer error messages, but its not something that is actionable to "would need to know about during an upgrade"

Reply:
yup, will do. wasnt sure what should be documented or not, so erred on the side of writing more